menas/src/main/scala/za/co/absa/enceladus/menas/services/SchemaRegistryService.scala

```diff
@@ -69,8 +114,34 @@ class SchemaRegistryService @Autowired()() {
    */
   def loadSchemaByUrl(remoteUrl: String): SchemaResponse = {
     try {
+      if (keyManagerFactory.isEmpty) {
+        logger.warn("keyManagerFactory is not, secure schema registry integration may not function properly. " +
+          s"Set ${SecureConfig.Keys.javaxNetSslKeyStore} and ${SecureConfig.Keys.javaxNetSslKeyStorePassword} to fix the issue.")
+      }
+      if (trustManagerFactory.isEmpty) {
+        logger.warn("trustManagerFactory is not, secure schema registry integration may not function properly. " +
+          s"Set ${SecureConfig.Keys.javaxNetSslTrustStore} and ${SecureConfig.Keys.javaxNetSslTrustStorePassword} to fix the issue.")
+      }
+
+      val ctx = SSLContext.getInstance(defaultSslContextProtocol)
+      ctx.init(
+        keyManagerFactory.map(_.getKeyManagers).orNull,
+        trustManagerFactory.map(_.getTrustManagers).orNull,
+        null // scalastyle:ignore null - Java API
```

Comment:
This could be a piece of syntactic sugar to not use null.

```suggestion
        None.orNull
```

Reply:
Honestly, I don't know. `null` here is much more obvious and _not wrong_, because the API directly specifies it, I am a bit siding to keep it as is. How strong is your opinion on this?

Reply:
No strong feelings. I just think it looks better. Functions the same. If you do not like it, you don't like it.